styles/main.css

```diff
@@ -73,6 +75,8 @@ header nav ul {
     margin: 0;
     padding: 0;
     float: right;
+}
+header nav {
     overflow: auto;
 }

```

Comment:
The `overflow:auto` declaration needs to apply to the PARENT element of the floated element